lib/AnimationContainer.dart

```diff
@@ -0,0 +1,79 @@
+import 'dart:math';
+import 'package:flutter/material.dart';
+
+
+const _duration = Duration(milliseconds: 400);
```

Comment:
Comment: Is the underscore necessary? My assumption is that its only relevant for private methods/members in a class.